### PR TITLE
Update node-telegram-bot-api

### PR DESCRIPTION
**Pull request by karvaton**
Add 'thumb' (thumbnail) property to 'SendAudioOptions' interface according to Telegram Bot API documentation (https://core.telegram.org/bots/api#sendaudio)

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -195,6 +195,7 @@ declare namespace TelegramBot {
         duration?: number | undefined;
         performer?: string | undefined;
         title?: string | undefined;
+        thumb?: string| Stream | Buffer;
```
Comment (A7med3bdulBaset) [inline types/node-telegram-bot-api/index.d.ts:198]: just formatting
```suggestion
        thumb?: string | Stream | Buffer;
```

Comment (A7med3bdulBaset) [inline types/node-telegram-bot-api/index.d.ts:198]: formatting
```suggestion
        thumb?: string | Stream | Buffer;
```

**Review by AhmedBaset — CHANGES_REQUESTED** [review]

**Review by AhmedBaset — COMMENTED** [review]
Can you please add `thumbnail` to the other interfaces e.g. document, animation, video, etc... as well

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/node-telegram-bot-api/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cb009a22267ea4a06502b2b21ede489443159835
</p>

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by karvaton** [comment]
Ready to merge

**Pull request merged**